qa/tasks/cephfs/test_volumes.py

```diff
@@ -7969,3 +7969,51 @@ def test_volumes_module_finisher_thread(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+class TestEmptyStringForCreates(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def _fs_cmd(self, *args):
+        return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
```

Comment:
you can use `run_ceph_cmd()` to eliminate need of creating this helper or it might be a good idea to create this in `class CephFSTestCase` so that this is usable to execute fs cmds at other places too?

Reply:
I hope its okay to take this in as is ?

Reply:
im fine with it